src/content/en/tools/chrome-devtools/evaluate-performance/_shared/set-up.md

```diff
@@ -0,0 +1,69 @@
+## Set up {: #set-up }
```

Comment:
This section is in a separate file because another tutorial is going to re-use it.